twittermap/conf/application.conf

```diff
@@ -78,5 +78,5 @@ bounded-mailbox {
 # for the debug purpose, we can load a smaller json to speed up the front-end
 cloudberry.register = "http://localhost:9000/admin/register"
 cloudberry.ws = "ws://localhost:9000/ws"
-us.city.path = "/public/data/city.sample.json"
+us.city.path = "/public/data/city.json"
```

Comment:
@JavierJia mentioned that this needs to be reverted. @JavierJia: can you confirm?